akasha-master/src/arcasha/assistant/ui.html

```
@@ -0,0 +1,1473 @@
+            <div class="set-row">
+              <label data-i18n="set.orchestration">参加モデル数</label>
+              <div style="display:flex;align-items:center;gap:12px">
+                <input type="range" id="setCount" min="1" max="50" step="1" value="2" />
+                <span class="count-val" id="setCountVal">2</span>
+              </div>
+              <div class="desc" data-i18n="set.orchestrationDesc">1 = Flash のみ / 2 = Flash + Pro（既定）/ 3〜50 = 推論ノードを増やしてフォールバックチェーンを拡張。タスク分類で空応答時に次のモデルへ委譲します。</div>
+            </div>
+            <div class="set-row">
+              <label data-i18n="set.thinking">💭 思考（Thinking）に使用するトークン上限</label>
+              <div style="display:flex;align-items:center;gap:12px">
+                <input type="range" id="setThinking" min="512" max="32768" step="200" value="8000" />
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)